bin/install_grass.sh

```diff
@@ -50,24 +48,27 @@ fi
 USER_HOME="/home/$USER_NAME"
 
 TMP_DIR=/tmp/build_grass
-mkdir "$TMP_DIR"
+mkdir -p "$TMP_DIR"
 
+# https://launchpad.net/~ubuntugis/+archive/ubuntu/ubuntugis-unstable
+apt-get --quiet --yes install software-properties-common
+add-apt-repository --yes ppa:ubuntugis/ubuntugis-unstable
```

Comment:
We are not mixing the UbuntuGIS ppa with OSGeoLive ppa, please remove.
The GRASS 8.4.0 is going to be provided in the OSGeoLive ppa

Reply:
I see, removed in 5a753a6040df7ac08b1654dd105a16026a19bf01